tutorials/dataframe/df021_createTGraph.C

```diff
@@ -0,0 +1,33 @@
+/// \file
+/// \ingroup tutorial_dataframe
+/// \notebook
+/// This tutorial shows how to fill a TGraph using the Dataframe.
+///
+/// \macro_code
+///
+/// \date July 2018
+/// \author Enrico Guiraud, Danilo Piparo, Massimo Tumolo
+
+
+
+void df021_createTGraph()
+{
+   std::vector<int> source(160);
+   for (int i = 0; i < 160; ++i)
+      source[i] = i;
+
+   ROOT::RDataFrame d(160);
+   auto dd = d.DefineSlotEntry("x1",
+                               [&source](unsigned int slot, ULong64_t entry) {
+                                  (void)slot;
+                                  return source[entry];
+                               })
+                .DefineSlotEntry("x2", [&source](unsigned int slot, ULong64_t entry) {
+                   (void)slot;
+                   return source[entry];
```

Comment:
Here an idea could be to use something like a trigonometric function to make a trend which is not as obvious as a straight line. Why not a square to see a parabola?